### PR TITLE
[pfcwd]: Assign different names to executors.

### PR DESCRIPTION
**Pull request by marian-pritsak**
Having executors with same names results in only first being registered.

Signed-off-by: marian-pritsak <marianp@mellanox.com>

### CODE REVIEW

File: orchagent/pfcwdorch.cpp [hunk 1]
```diff
@@ -35,7 +35,7 @@ PfcWdOrch<DropHandler, ForwardHandler>::PfcWdOrch(DBConnector *db, vector<string
     auto interv = timespec { .tv_sec = PFC_WD_LOSSY_POLL_TIMEOUT_SEC, .tv_nsec = 0 };
     auto timer = new SelectableTimer(interv);
     auto executor = new ExecutableTimer(timer, this);
-    Orch::addExecutor("", executor);
+    Orch::addExecutor("COUNTERS_POLL", executor);
```
Comment (sihuihan88) [inline orchagent/pfcwdorch.cpp]: change to more specific name?

Comment (lguohan) [reply orchagent/pfcwdorch.cpp]: any suggestion?

Comment (sihuihan88) [reply orchagent/pfcwdorch.cpp]: PFC_WD_LOSSY?

Comment (marian-pritsak) [reply orchagent/pfcwdorch.cpp]: This executor will be extended with more counters (e.g. for multicast queues). I'd want to keep it more general.

**Pull request merged**